src/Conversation.php

```diff
@@ -310,4 +310,9 @@ public function getSaveActions()
     {
         return $this->save_actions;
     }
+
+    public function jsonserialize()
+    {
+        return $this->toArray;
```

Comment:
Je t'avais dit de retourner le résultat de la fonction toArray et non la fonction elle même...
`return $this->toArray();`